Podcast/DiscoverTableViewCell.swift

```diff
@@ -125,6 +128,10 @@ class DiscoverTableViewCell: UITableViewCell {
         contentView.addSubview(clickToPlayImageButton)
         
         adjustForScreenSize()
+        
+        heightConstraint = NSLayoutConstraint(item: self.contentView, attribute: NSLayoutAttribute.height, relatedBy: NSLayoutRelation.equal, toItem: nil, attribute: NSLayoutAttribute.notAnAttribute, multiplier: 1, constant: height)
+    
+        self.contentView.addConstraint(heightConstraint!)
```

Comment:
Should use `NSLayoutConstraint.activate(_:)` instead as NSView's `addConstraint(_:)` is soft deprecated.